packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -52,15 +53,9 @@ function Spec(attrs) {
     function() {
       return '';
     };
-  this.expectationResultFactory = attrs.expectationResultFactory ||
-    function() {};
   this.queueRunnerFactory = attrs.queueRunnerFactory || function() {};
   this.throwOnExpectationFailure = !!attrs.throwOnExpectationFailure;
 
-  if (!this.queueableFn.fn) {
-    this.pend();
```

Comment:
`queueableFn` always has a `fn`.  I had erased it to a note to myself, but it looks like I forgot to add it back.  Shall I make a PR to undo that?